store/tikv/region_cache.go

```diff
@@ -275,9 +275,10 @@ type RegionCache struct {
 	enableForwarding bool
 
 	mu struct {
-		sync.RWMutex                         // mutex protect cached region
-		regions      map[RegionVerID]*Region // cached regions are organized as regionVerID to region ref mapping
-		sorted       *btree.BTree            // cache regions are organized as sorted key to region ref mapping
+		sync.RWMutex                          // mutex protect cached region
+		regions      map[RegionVerID]*Region  // cached regions are organized as regionVerID to region ref mapping
+		versions     map[uint64][]RegionVerID // cache regionID to RegionVerID mapping
```

Comment:
how about just maintain `latestVersions` (i.e. regionID -> latest ver mapping) instead of regionID -> []RegionVerID mapping ~?

we can only maintain lastest ver instead of ver array if the semantic for `LocateRegionByID` is "find latest region info for region id"